skribenten-backend/src/test/kotlin/no/nav/pensjon/brev/skribenten/db/BrevredigeringServiceTest.kt

```diff
@@ -94,13 +107,46 @@ class BrevredigeringServiceTest {
 
     @Test
     fun `can update brevredigering`() = runBlocking {
+        coEvery { penService.hentPesysBrevdata(any(), any(), any()) } returns
```

Comment:
Kanskje mocke med argument-matchers for sak og brevkode?
`penService.hentPesysBrevData(any(), eq(sak.saksId), eq(Brevkode.Redigerbar.INFORMASJON_OM_SAKSBEHANDLINGSTID)`